.github/workflows/push-build-image-indexer.yaml

```diff
@@ -6,8 +6,21 @@ on:
       - "main"
       - "release-*"
     paths-ignore:
+      - ".github/**"
+      - ".reuse/**"
+      - "LICENSES/**"
+      - "config/**"
+      - "data/**"
       - "docs/**"
+      - "scripts/**"
+      - "tests/**"
+      - "**/*.md"
       - "src/**"
+      - "doc_indexer/.vscode/**"
+      - "doc_indexer/tests/**"
+      - CODEOWNERS
+      - LICENSE
+      - "sec-scanners-config.yaml"
```

Comment:
You can instead do: [example](https://github.com/kyma-project/kyma-companion/blob/5b78d67f575d7fa64102cc89d4521c93637f7c4a/.github/workflows/pull-tests-doc-indexer.yaml#L10)

<img width="578" height="193" alt="Image" src="https://github.com/user-attachments/assets/1a264dd4-5a52-41b1-8ca6-70faaf82fefb" />